### DISCUSSION TITLE
Configuration file options do not take effect

### DISCUSSION THREAD

**Discussion by lzcd** [ghostty-org/ghostty#3846] [Q&A] [answered]
I have created a configuration file but only the background colour seems to take effect.
It does not seem to matter what configuration options I choose; none of the others take effect.
The configuration file was created via the Ghostty 'Settings' menu item.
(and thus, being the mac version, was created at `/Users/luke/Library/Application\ Support/com.mitchellh.ghostty/config`)

Here is the configuration:
```
background = 000000
font-size = 16
font-feature = -liga
theme = ayu
```

Here are the Ghostty version details:
```
Version
  - version: 1.0.0
  - channel: stable
Build Config
  - Zig version: 0.13.0
  - build mode : builtin.OptimizeMode.ReleaseFast
  - app runtime: apprt.Runtime.none
  - font engine: font.main.Backend.coretext
  - renderer   : renderer.Metal
  - libxev     : main.Backend.kqueue
```

Any ideas, suggestions etc?

**Comment by jmckillican** [comment]
are you quitting Ghostty after you make changes?  some changes need an application restart.

**Reply by lzcd** [reply]
I stand corrected.
(After restarting) Background and font size take effect.
Ligatures and themes do not.

**Reply by jmckillican** [reply]
for the theme, you need to copy the contents of the theme file from 

`/Applications/Ghostty.app/Contents/Resources/ghostty/themes/ayu`

and paste that in your ghostty config file.

**Reply by lzcd** [reply]
Cool.
That takes care of theme.

Any ideas on how to disable ligatures?

**Reply by 00-kat** [reply]
https://ghostty.org/docs/config/reference#font-feature

**Reply by jmckillican** [reply, marked as answer]
try disabling all types of ligatures in case:

```
font-feature = -calt
font-feature = -dlig
font-feature = -liga
```

**Reply by lzcd** [reply]
Thanks.

`-calt` did the trick.

**Comment by deskangel** [comment]
I encountered the same issue on Mac. After converting the line endings to Windows format, it worked.

You can run `ghostty +show-config` to check if the configurations are messed up.
